tigerpath/majors_and_certificates/majors/REL.json

```diff
@@ -32,169 +32,344 @@
       ]
     },
     {
-      "name": "Subfields",
-      "max_counted": 1,
-      "min_needed": "ALL",
-      "explanation": "Students will select at least one course from each of the five subfields of the faculty: Islam, Religion and Critical Thought, Religion in America, Religions of Asia, Religions of the Ancient Mediterranean.",
+      "name":"Departmentals",
+      "max_counted":null,
+      "min_needed":"ALL",
       "req_list": [
         {
-          "name": "Islam",
+          "name": "Streams",

```

Comment:
https://religion.princeton.edu/undergraduate-program/major-requirements/

Requirement 1
Concentrators are required to complete at least nine departmental courses by the end of the senior year. Courses taken prior to declaring a Religion concentration count toward this total.

Undergraduate courses in the Department of Religion are classified in two ways: 1) Traditions and 2) Themes. Concentrators must take at least three courses in either a particular tradition in a Traditions stream or a particular theme in a Thematic stream.

Students will choose one of the two streams in conversation with the Director of Undergraduate Studies during their Junior Fall.